test/gdUnit4Test.csproj

```diff
@@ -31,4 +31,8 @@
     <ProjectReference Include="..\api\gdUnit4Api.csproj" />
     <ProjectReference Include="..\testadapter\gdUnit4TestAdapter.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="src\core\resources\scenes\TestSceneWithInitialization.tscn" />
```

Comment:
If there is a reason to add the scene to the project?
If not than please remove this change

Reply:
Oops that was mistakenly added

Reply:
please remove it ;)